### ISSUE TITLE
There is no route for the path: /

### ISSUE THREAD

**Issue by guivho** [MaksTr/meteor-mantra-kickstarter#16]
Hi,

When I'm running the app, I get a blank screen in the browser and the console complains:

'There is no route for the path: /'

When I uncollapse this message, this backtrace appears:

`
Router.prototype._notfoundRoute() kadira_flow-router.js:2206
Router.prototype._updateCallbacks/<() kadira_flow-router.js:2394
[3]</</Route.prototype.middleware/<() kadira_flow-router.js:607
nextEnter() kadira_flow-router.js:439
[3]</</page.dispatch() kadira_flow-router.js:445
[3]</</page.replace() kadira_flow-router.js:408
Router.prototype.initialize/</self._page[fnName]() kadira_flow-router.js:2241
[3]</</page.start() kadira_flow-router.js:301
page() kadira_flow-router.js:235
Router.prototype.initialize() kadira_flow-router.js:2250
<anonymous> kadira_flow-router.js:2691
maybeReady() meteor.js:825
loadingCompleted() meteor.js:831
`

Note that I experience the same problem with the 'Impossible' app from Ken Roger's new book.

So this must be some general problem related to the environment rather than the app.

Anybody any ideas?

TIA, Guido

**Comment by rgstephens** [comment]
Along with the above error, I'm also getting:

> Uncaught Error: Cannot find module 'react-simple-di'

**Comment by rts100** [comment]
I had the same error and a subsequent "Uncaught Error: Cannot find module 'react-komposer'."
The errors stopped when I npm installed the react-simple-di and react-komposer modules.

**Comment by vonwao** [comment]
Yeah, it's still missing react-simple-di and react-komposer.  You can add manually npm i --save ...

(the _users module hasn't been updated to use mantra-core dependency)
